packages/nodes-base/nodes/HackerNews/HackerNews.node.ts

```diff
@@ -0,0 +1,176 @@
+import { IExecuteFunctions } from 'n8n-core';
+import {
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription,
+	IDataObject,
+} from 'n8n-workflow';
+import { OptionsWithUri } from 'request-promise-native';
+
+export class HackerNews implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Hacker News',
+		name: 'hackerNews',
+		icon: 'file:hackernews.png',
+		group: ['transform'],
+		version: 1,
+		description: 'Consume Hacker News API',
+		defaults: {
+			name: 'Hacker News',
+			color: '#772244',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		properties: [
+			// ----------------------------------
+			//         Operation
```

Comment:
Nice commenting!! 😄

Reply:
Thanks! I did it mostly to avoid losing my place.